data/circulation_policies.json

```diff
@@ -9,9 +9,21 @@
     "allow_checkout": true,
     "checkout_duration": 30,
     "allow_requests": true,
-    "number_of_days_before_due_date": 5,
-    "number_of_days_after_due_date": 5,
-    "reminder_fee_amount": 2.0,
+    "reminders": [
+      {
+        "type": "due_soon",
+        "days_delay": 5,
+        "communication_channel": "email",
+        "template": "email/due_soon"
```

Comment:
maybe email could be remove and use only `due_soon`, `overdue` or other template name.
it's possible to define template path depending of communication_language

Reply:
`+ 1` by language by library is possible later

Reply:
for language, this is not the library language but the patron "communication_language" that is used.
So for this point, it isn't the CIPO to choose which template_language_file to use. 
So the actual solution seems woking fine : in `template` we specified the directory where to find the correct `template_language_file.txt`